sirmordred/task_panels.py

```diff
@@ -237,13 +237,16 @@ def __configure_kibiter_setting(self, endpoint, data_value=None):
         kibana_url = self.conf['panels']['kibiter_url'] + KIBANA_SETTINGS_URL
         endpoint_url = kibana_url + '/' + endpoint
 
-        res = self.grimoire_con.post(endpoint_url, headers=kibana_headers,
-                                     data=json.dumps(data_value), verify=False)
         try:
+            res = self.grimoire_con.post(endpoint_url, headers=kibana_headers,
+                                         data=json.dumps(data_value), verify=False)
             res.raise_for_status()
         except requests.exceptions.HTTPError:
             logger.error("Impossible to set %s: %s", endpoint, str(res.json()))
             return False
+        except requests.exceptions.ConnectionError:
+            logger.error("Impossible to connect to kibiter %s", self.conf['panels']['kibiter_url'])
```

Comment:
WRT your first comment, I included a method `anonymize_url`, which removes credentials info from an URL. The method is the same of the one in [ELK](https://github.com/chaoss/grimoirelab-elk/blob/master/grimoire_elk/elastic.py#L149). This solution is OK, but duplicates code. Other solutions could be either (i) to move the method `anonymize_url` to grimoirelab-toolkit and refactor the code of ELK and mordred accordingly or (ii) don't print the `kibiter_url`in the log message, but reference the param in the setup.cfg. (i) requires some work since that method is extensively used in ELK, (ii) requires the user to check the `setup.cfg`.

Let me know if you have other solutions in mind or which solution you prefer @sduenas